dataland-frontend/tests/e2e/fixtures/common/DataPointFixtures.ts

```diff
@@ -1,5 +1,10 @@
 import { faker } from "@faker-js/faker";
-import { CompanyReportReference, DataPointBigDecimal, DataPointYesNo, QualityOptions } from "@clients/backend";
+import {
+  CompanyReportReference,
```

Comment:
The method `generateNumericOrEmptyDatapoint` is unnecessary.
You can directly use its return statement wherever you need this: `valueOrUndefined(generateDatapoint(valueOrNull(faker.number.int()), reports));`